packages/manager/modules/octavia-load-balancer/src/loadbalancers/loadbalancers.html

```diff
@@ -0,0 +1,98 @@
+<div data-ui-view>
+    <div class="octavia-load-balancers">
+        <oui-header
+            data-heading="{{ :: 'octavia_load_balancers_title' | translate }}"
+        ></oui-header>
+
+        <div data-ovh-alert="octavia.alerts.global"></div>
+        <oui-datagrid
+            class="table-striped"
+            data-page-size="{{ $ctrl.paginationSize }}"
+            data-rows="$ctrl.resources"
+            data-criteria="$ctrl.criteria"
```

Comment:
A lot of this attribute are using undefined property (ex: criteria) and can be safely removed

Reply:
Done

Reply:
Done